property-services/src/main/java/org/egov/pt/models/user/User.java

```diff
@@ -68,11 +68,12 @@
         @JsonProperty("gender")
         private String gender;
 
-        @Pattern(regexp = "(^[6-9][0-9]{9}$)", message = "Inavlid mobile number, should start with 6-9 and contain ten digits of 0-9")
+        //@Pattern(regexp = "(^[6-9][0-9]{9}$)", message = "Inavlid mobile number, should start with 6-9 and contain ten digits of 0-9")
         @NotNull
         @SafeHtml
         @JsonProperty("mobileNumber")
-        private String mobileNumber;
```

Comment:
Do not change the data type of mobile number.
There will be new variable for alternate mobile number
If we store every mobile number in a single list.
Then we cannot differentiate between which number belongs to owner and which number belongs to secondary person